app/javascript/components/upload/ExpandableFileForm.jsx

```diff
@@ -42,16 +47,16 @@ export default function ExpandableFileForm({
               <FontAwesomeIcon icon={expanded ? faChevronUp : faChevronDown} />
             </button>
           </div>
-          <div className="flexbox">
+          {!doNotShowFileUploadControls && <div className="flexbox">
```

Comment:
See related comments.
```suggestion
          {isUploadEnabled && <div className="flexbox">
```